oozie-to-airflow/o2a.py

```diff
@@ -36,6 +39,25 @@ def main():
     if not dag_name:
         dag_name = os.path.basename(input_directory_path)
 
+    conf_path = os.path.join(input_directory_path, CONFIGURATION_PROPERTIES)
+    if not os.path.exists(conf_path):
```

Comment:
You can use https://docs.python.org/3/library/os.path.html#os.path.isfile

Reply:
Done